src/main/resources/ApplicationResources_es.properties

```diff
@@ -456,6 +456,9 @@ admin.config.setup3.welcome=¡Felicitaciones\! Ha instalado satisfactoriamente s
 admin.config.archivalMode=Modo Archivo
 admin.config.archivalMode.help=Por favor seleccione esta casilla si desea que su IPT archive todas las versiones publicadas para todos los recursos. Esto permitirá realizar un seguimiento al historial de versiones de un recurso. Si está deseleccionada, las versiones anteriores serán reemplazadas por la última. Tenga cuidado de no quedarse sin espacio en el disco.
 admin.config.archivalMode.error=Modo Archivo inválido (debe ser verdadero or falso)
+admin.config.archivalLimit=Number of archived versions (require Archival Mode)
```

Comment:
You don't need to do this. CrowdIn will automatically copy the English properties into the translatable files.

We need to check if this breaks the CrowdIn functionality